rskj-core/src/test/java/co/rsk/util/CacheElementTest.java

```diff
@@ -0,0 +1,22 @@
+package co.rsk.util;
+
+import org.junit.Test;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.awaitility.Awaitility.await;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class CacheElementTest {
+
+    @Test
+    public void hasExpired_OK() {
```

Comment:
is the "_" usage in naming something we do everywhere? or only in Bridge stuff? (I've only seen it there)